docs/components/Playground/Playground.js

```diff
@@ -0,0 +1,60 @@
+import { Sandpack } from "@codesandbox/sandpack-react";
+
+import nextCloudinaryPkg from '../../../next-cloudinary/package.json';
+import nextCloudinary from "../../internals/next-cloudinary?raw";
+
+const nextConfgJs = `
+process.env.NEXT_PUBLIC_CLOUDINARY_CLOUD_NAME = "${process.env.NEXT_PUBLIC_CLOUDINARY_CLOUD_NAME}";
+
+module.exports = {
+  reactStrictMode: true
+};
+`
+
+const styleCss = `
+img {
+  max-width: 100%;
+  height: auto;
+}
+`;
+
+const Playground = (props) => {
+  const { children, filename } = props;
+  return (
+    <Sandpack
```

Comment:
Hey, Sandpack is not rendering correctly on SSR, so I'd suggest taking a look at this [guide](https://sandpack.codesandbox.io/docs/guides/ssr)